### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by ghost** [rafaelvcaetano/melonDS-android#466] [General]
How about having one? 
So people could be up to date on news and ask questions if needed.

**Comment by HJMBgit84** [comment]
I like your idea, hope Rafael will consider it for feedback and performance testing 😊👍🏻

**Comment by rafaelvcaetano** [comment]
I've thought about it but I don't really see any way I could keep it interesting. I don't really like to share details of what I'm working on because if I don't manage to finish the features, it will just bring disappointment. And regarding questions, discussions already address this and they are more accessible to other users since they don't get lost in an endless chat.

I wouldn't mind having one just for more direct contact with you guys, but, yeah, I wouldn't really post a lot apart from answering questions or participating in casual conversation. Plus, I don't really have the time to moderate it, so I would need someone to help with that.

**Reply by HJMBgit84** [reply]
It is ok and understandable, it would be good for eventual conversations and have a bit of contact, thanks even if you create it or not, your job is appreciated, thanks a lot for bringing us Melon in android 😊
